tests/tools/test_dist_part.py

```diff
@@ -195,25 +149,72 @@ def rand_edges(num_src, num_dst, num_edges):
 
-        all_etypes = ['affiliated_with', 'writes', 'cites', 'rev_writes']
         for etype in all_etypes:
             assert len(meta_data['edge_map'][etype]) == num_parts
         assert meta_data['etypes'].keys() == set(all_etypes)
         assert meta_data['graph_name'] == 'mag240m'
 
-        all_ntypes = ['author', 'institution', 'paper']
         for ntype in all_ntypes:
             assert len(meta_data['node_map'][ntype]) == num_parts
         assert meta_data['ntypes'].keys() == set(all_ntypes)
         assert meta_data['num_edges'] == g.num_edges()
         assert meta_data['num_nodes'] == g.num_nodes()
         assert meta_data['num_parts'] == num_parts
 
+        edge_dict = {}
+        edge_data_gold = {}
+
+        if include_edge_data:
+            # Create Id Map here.
+            num_edges = 0
+            for utype, etype, vtype in g.canonical_etypes:
+                fname = ':'.join([utype, etype, vtype])
+                edge_dict[fname] = np.array(
+                    [num_edges, num_edges + g.number_of_edges(etype)]
+                ).reshape(1, 2)
+                num_edges += g.number_of_edges(etype)
+
+            assert num_edges == g.number_of_edges()
+            id_map = dgl.distributed.id_map.IdMap(edge_dict)
+            orig_etype_id, orig_type_eid = id_map(np.arange(num_edges))
+
+            # check edge_data
+            num_edges = {
+                'paper:cites:paper': num_cite_edges,
+                'author:writes:paper': num_write_edges,
+                'paper:rev_writes:author': num_write_edges,
+            }
+            output_dir = os.path.join(root_dir, 'chunked-data')
+            output_edge_data_dir = os.path.join(output_dir, 'edge_data')
+            for etype, feat in [
+                ['paper:cites:paper', 'count'],
+                ['author:writes:paper', 'year'],
+                ['paper:rev_writes:author', 'year'],
+            ]:
+                output_edge_sub_dir = os.path.join(output_edge_data_dir, etype)
+                features = []
+                for i in range(num_chunks):
+                    chunk_f_name = '{}-{}.npy'.format(feat, i)
+                    chunk_f_name = os.path.join(
+                        output_edge_sub_dir, chunk_f_name
+                    )
+                    assert os.path.isfile(chunk_f_name)
+                    feat_array = np.load(chunk_f_name)
+                    assert feat_array.shape[0] == num_edges[etype] // num_chunks
+                features.append(feat_array)
+                edge_data_gold[etype + '/' + feat] = np.concatenate(features)
+
         for i in range(num_parts):
             sub_dir = 'part-' + str(i)
-            assert meta_data[sub_dir]['node_feats'] == 'part{}/node_feat.dgl'.format(i)
-            assert meta_data[sub_dir]['edge_feats'] == 'part{}/edge_feat.dgl'.format(i)
-            assert meta_data[sub_dir]['part_graph'] == 'part{}/graph.dgl'.format(i)
+            assert meta_data[sub_dir][
```

Comment:
this looks a bit weird,it's formatted by `black`?

Reply:
This is the output from black. There is not much we can do here.